src/main/webapp/app/modules/managerSurveysDashboard/managerSurveysDashboard.tsx

```diff
@@ -99,22 +99,19 @@ const ManagerSurveysDashboard = () => {
             </Dropdown.Item>
           </Dropdown.Menu>
         </Dropdown>
-        <Link to={surveyPath}>
-          <div className={'name-row'}>
```

Comment:
The question is why is this deleted? The result is there are no names on survey boxes, which looks bad. You can just extract it from the Link tag, but it must stay. 

My idea is to duplicate the logic the Status option from dropdown - when you click it, it should navigate to the status subpage.